### PR TITLE
Feat: Conferences

### PR DESCRIPTION
**Pull request by vladamx**
Added ability for us to share available conferences. Additionally, I fixed some build issues.

### CODE REVIEW

File: client/conferences.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+// This file is built with parcel-bundler locally.
+// Dist is static/js
+import moment from 'moment'
+import lodash from 'lodash'
+
+/*  eslint-env jquery */
+
+function onLoadEvent () {
+  console.log('loaded')
```
Comment (simalexan) [inline client/conferences.js:9]: all good so far, if possible please remove this `console.log`

File: bin/server.js [hunk 2]
```diff
@@ -17,14 +19,9 @@ function start () {
   })
 
   http.createServer((req, res) => mount(req, res)).listen(8080, () => {
```
Comment (simalexan) [inline bin/server.js:21]: thank you, all looks good, just maybe change this fixed port number as well

File: scripts/helpers/strip.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+/**
+ *
+ * @param {string} path
+ */
+module.exports = function(path, options) {
+    return path.replace('/index.html', "")
```
Comment (simalexan) [inline scripts/helpers/strip.js:8]: are you sure you wanted to use double quotes here?

File: package.json [hunk 4]
```diff
@@ -4,8 +4,12 @@
...
   "main": "bin/build.js",
   "scripts": {
-    "build": "./bin/build.js",
-    "start": "standard --verbose | snazzy && ./bin/server.js"
+    "clean": "rimraf ./node_modules",
+    "lint": "standard --verbose | snazzy",
+    "commit": "npm run lint && git-cz",
+    "build": "npm run lint && node ./bin/build.js",
+    "server": "(wait-on http://localhost:$PORT && opn http://localhost:$PORT) | node ./bin/server.js",
+    "start": "npm run lint && cross-env PORT=8080 npm run server"
```
Comment (simalexan) [inline package.json:12]: thanks, looks good

**Review by simalexan — COMMENTED** [review]
Thank you very much for the effort and time! It's really appreciated! This is an initial code-wise check :) after this one, we'll test the functionality

File: scripts/helpers/strip.js (hunk 3 above)
Comment (vladamx) [reply scripts/helpers/strip.js:8]: Maybe i was daydreaming :). You have very strong attention to detail. I don't know why my tools didn't catch that. :(

File: bin/server.js (hunk 2 above)
Comment (vladamx) [reply bin/server.js:21]: Missed that! Thanks.

File: client/conferences.js (hunk 1 above)
Comment (vladamx) [reply client/conferences.js:9]: Of course. Again, I need to fix the tools. This can be a serious performance problem, also.

Comment (vladamx) [reply client/conferences.js:9]: "serious"

**Pull request merged**